frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```
@@ -64,6 +64,17 @@
   "What is a Namespace Policy?": "What is a Namespace Policy?",
   "Namespace policy can be set to one single read and write source, multi read sources or cached policy.": "Namespace policy can be set to one single read and write source, multi read sources or cached policy.",
   "Namespace Policy Type": "Namespace Policy Type",
+  "What is Caching ?": "What is Caching ?",
+  "Caching is a policy which creates local copies of the data. It saves the copies locally to improve performance for frequently accessed data. Each cached copy has a TTL and is verified against the hub. Each non read operation(upload, overwrite, delete) is performed on the hub": "Caching is a policy which creates local copies of the data. It saves the copies locally to improve performance for frequently accessed data. Each cached copy has a TTL and is verified against the hub. Each non read operation(upload, overwrite, delete) is performed on the hub",
+  "Hub namespace store ": "Hub namespace store ",
+  "A single namespace-store, defines the read and write target of the namespace bucket": "A single namespace-store, defines the read and write target of the namespace bucket",
+  "Namespace store": "Namespace store",
+  "Cache data settings": "Cache data settings",
+  "The data will be temporarily copied on a backing store in order to later access it much more quickly.": "The data will be temporarily copied on a backing store in order to later access it much more quickly.",
+  "Backing store": "Backing store",
+  "Local BackingStore is recommmended for better performance.": "Local BackingStore is recommmended for better performance.",
```

In some places, we are ending with a period and in some, we are not.
```suggestion
  "Local BackingStore is recommended for better performance": "Local BackingStore is recommended for better performance",
```

---

fixed